include/herokuish.bash

```diff
@@ -92,22 +92,23 @@ randomize-unprivileged() {
 herokuish-test() {
 	declare desc="Test running an app through Herokuish"
 	declare path="${1:-/}" expected="$2"
-	export PORT=5678
+	PORT=$(awk 'BEGIN{ srand();print int(rand()*(15600-1000))+1000 }')
```

Comment:
Why the awk change?

Reply:
randomizes the test port. i was seeing some oddities in 2.0 where our `nc` check was succeeding but the listener wasn't actually ready. my hypothesis is that kernel level options are different and perhaps the old port isn't cleaned up right away.